superset-frontend/plugins/legacy-plugin-chart-world-map/src/index.js

```diff
@@ -45,6 +45,13 @@ const metadata = new ChartMetadata({
   ],
   thumbnail,
   useLegacyApi: true,
+  label: {
+    name: t('verified'),
+    description: t(
+      'This chart was tested and verified, so the overall experience should be stable.',
```

Comment:
I understand that a label is very flexible, but do we actually want such high flexibility? If this were, say, an enum of `VERIFIED`, `DEPRECATED`, ... , it could be used for more programmatic things. For example, rather than using a searchWeight that has to be the same for all verified charts, the logic in the gallery's sort function could sort by the value of this enum. And having this text duplicated in every `verified` plugin doesn't really seem ideal to me.

We already have a boolean field `deprecated`, so if we did switch this to an enum we would probably want to... deprecate the deprecated field.

Reply:
Thanks for this inputs！This makes sense. @rusackas I think we can unify it into an enum and influence the search results based on the definition of this field.